examples/diesel/Cargo.toml

```diff
@@ -25,4 +26,4 @@ serde_derive = "1.0"
 
 [dev-dependencies]
 diesel_migrations = "1.4.0"
-tokio = "0.1"
+tokio = { version = "0.2.6", features = ["full"] }
```

Comment:
What specifically do we need from the `full` feature that is missing from the default?

Reply:
Tokio includes [nothing by default][0] 😞.  You have to specify what features you need explicitly.  I believe we use:
- blocking
- net
- dns
- rt-core
- time
- fs (maybe?)

But I haven't tried to be more granular than just full.

[0]: https://github.com/tokio-rs/tokio/blob/eb1a8e1792b2c4b296be47a0681421c90bbdbf7a/tokio/Cargo.toml#L28